src/components/intro-section.tsx

```
@@ -0,0 +1,220 @@
+            onAnimationComplete={() => {
+              requestAnimationFrame(() => {
+                setPhase("interactive");
+                onComplete?.();
+              });
+            }}
+            className="relative z-10 w-full max-w-5xl px-4"
+          >
+            <svg
+              viewBox="0 0 800 200"
+              className="w-full h-auto overflow-visible pointer-events-none select-none"
+            >
```

Biome (2.3.13)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)